plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergFileWriterFactory.java

```diff
@@ -146,19 +147,21 @@ public IcebergFileWriter createPositionDeleteWriter(
             ConnectorSession session,
             HdfsContext hdfsContext,
             IcebergFileFormat fileFormat,
-            Map<String, String> storageProperties)
+            Map<String, String> storageProperties,
+            Optional<DataSize> stringStatisticsLimit)
     {
         switch (fileFormat) {
             case PARQUET:
-                return createParquetWriter(outputPath, POSITION_DELETE_SCHEMA, jobConf, session, hdfsContext);
+                return createParquetWriter(FULL_METRICS_CONFIG, outputPath, POSITION_DELETE_SCHEMA, jobConf, session, hdfsContext);
             case ORC:
-                return createOrcWriter(FULL_METRICS_CONFIG, outputPath, POSITION_DELETE_SCHEMA, jobConf, session, storageProperties);
+                return createOrcWriter(FULL_METRICS_CONFIG, outputPath, POSITION_DELETE_SCHEMA, jobConf, session, storageProperties, stringStatisticsLimit);
```

Comment:
We never want to truncate the stats path, right? So could we do

`DataSize.ofBytes(Long.MAX_VALUE)` here?

Reply:
hmm that makes sense, I will change it

Reply:
Actually it has to be Integer.MAX_VALUE